docs/README.md

```diff
@@ -25,3 +25,11 @@ There are also some helper scripts:
 
  - `npm run lint` : Run eslint over the javascript files
  - `npm run i18n` : Create a PHP file with the strings from the javascript files, [used to get around lack of JS support in WordPress.org](https://github.com/WordPress/packages/tree/master/packages/i18n#build).
+
+## Dev Docs
+
+There is a "devdocs" page which is useful for displaying components individually outside of the application:
+
+```http://<your-wp-site>/wp-admin/admin.php?page=wc-admin#/devdocs```
+
+This is useful for viewing of (WooCommerce)[https://woocommerce.github.io/wc-admin/#/components/] components and ad-hoc testing.
```

Comment:
Looks like your link formatting is reversed, it should be `[]` around the text & `()` around the link. Could you also make the link text include the word "components"? that way it's a little more descriptive.

Reply:
Oops, will fix.

Reply:
Fixed in 093e36e. Thanks!